### PR TITLE
Fix Required #17: Install Node.js/python3 based on OS

### LINKED ISSUE #17: Add support for install of package
Add support for install of package like NodeJS, python3, pip3, etc. according to OS Selection

Like :- if OS Selection is Ubuntu and Language type is Nodejs
ADD Statement:-  RUN apt install nodejs npm -y

### PR DESCRIPTION
**Pull request by yrahul3910**

### CODE REVIEW

**Comment by BRAVO68WEB** [comment]
# Error :- undefined in generated Dockerfile

## How to Produce :-
image

## Result :- 
image

**Comment by BRAVO68WEB** [comment]
Please fix it!

**Comment by BRAVO68WEB** [comment]
@yrahul3910 can you fix it?

**Comment by yrahul3910** [comment]
Yup, will do!

**Comment by yrahul3910** [comment]
I've fixed the issue now!

**Comment by BRAVO68WEB** [comment]
Sir Still the Same issue 

I have attached the Dockerfile generated, please do check

```Dockerfile
FROM dftechs/debian-dev
COPY package.json .
COPY src .
RUN [ "sudo apt-get install -y --no-install-recommends undefined && npm install" ]
EXPOSE 3000
CMD [ "node", "index.js" ]
```

I think it should be "nodejs" in place of "undefined"

**Comment by yrahul3910** [comment]
@BRAVO68WEB Have you pulled the newer commit 2cc32fc? Here is what I get:

Screen Shot 2020-10-05 at 1 11 25 AM

**Comment by BRAVO68WEB** [comment]
Yes

**Comment by BRAVO68WEB** [comment]
Let me try again

**Comment by BRAVO68WEB** [comment]
Ok it's working 
I removed my old installations and installed Globally

then its working

**Comment by BRAVO68WEB** [comment]
Nice work ! @yrahul3910

**Comment by BRAVO68WEB** [comment]
@all-contributors please add @yrahul3910 for code
